react-charts/bar-chart/step2.md

```diff
@@ -1,24 +1,55 @@
-We'll begin by adding a CSS selector to define the chart size...
-
-- Katacoda is setting up a new React application for you. You'll be ready to code once the server starts and you can see "Welcome to PatternFly" on the lower pane.
-
-## Task
-
-1) Navigate to the `src` folder and open `src/app.css`{{open}}
-
-2) Next we'll add some CSS to define the overall chart size
-
-Copy the following code into the app.css file, replacing all of the content there:
-
-<pre class="file" data-filename="src/app.css" data-target="replace">
-.chart-container {
-  height: 250px;
-  width: 600px;
+Now we'll create a simple bar chart...
+
+## Tasks
+
+1) <strong>Navigate to the `src` folder and open `src/App.js`{{open}}</strong>
+
+2) <strong>Set up the general structure of the chart and the necessary import statements.</strong>
+
+Copy the following code into the `App.js` file, replacing all of the content there:
+
+<pre class="file" data-filename="App.js" data-target="replace">
+import React from 'react';
+import "@patternfly/react-core/dist/styles/base.css";
+import './app.css';
+import {
+  Chart,
+  ChartAxis,
+  ChartBar,
+  ChartGroup,
+  ChartThemeColor,
+  ChartVoronoiContainer
+} from '@patternfly/react-charts';
+
+function App() {
+  return (
+    &lt;div className=&quot;chart-container&quot;&gt;
+      &lt;Chart
+        domainPadding={{ x: [30, 25] }}
+        height={250}
+        width={600}
+      &gt;
+        &lt;ChartBar 
+          data={[
+            { name: &#39;Cats&#39;, x: &#39;2015&#39;, y: 1 }, 
+            { name: &#39;Cats&#39;, x: &#39;2016&#39;, y: 2 }, 
+            { name: &#39;Cats&#39;, x: &#39;2017&#39;, y: 5 }, 
+            { name: &#39;Cats&#39;, x: &#39;2018&#39;, y: 3 }
+          ]} 
+        /&gt;
+      &lt;/Chart&gt;
+    &lt;/div&gt;
+  );
 }
+
+export default App;
```

Comment:
```suggestion
export default `App`;
```

Reply:
I don't understand this suggestion. It shouldn't probably be there.

Reply:
yep don't implement